assets/keymaps/vim.json

```diff
@@ -532,6 +543,18 @@
       ]
     }
   },
+  {
+    "context": "Editor && vim_mode == normal",
+    "bindings": {
+      "g c c": "editor::ToggleComments"
```

Comment:
oh,it is this

Reply:
https://github.com/zed-industries/zed/assets/19868292/b78e8bf4-da6f-4fcb-9384-49169cc3016f

Reply:
Oh, you want to go back to normal mode?

Reply:
> Oh, you want to go back to normal mode?

at least in vscode.